salt/states/virt.py

```diff
@@ -407,93 +590,51 @@ def running(name,
                     address: 192.168.0.125
 
     '''
-
-    ret = {'name': name,
-           'changes': {},
-           'result': True,
-           'comment': '{0} is running'.format(name)
-           }
-
-    try:
+    if update:
```

Comment:
It seems that someone still cannot use the `update` kwarg to change behavior if i'm reading it correctly. What if they want to ensure an update does not occur? This would be changing behavior without proper deprecation still. Can you elaborate why you are changing this behavior?

Reply:
If using the `update` parameter to ensure an update doesn't occur, this means that users have to set the deprecated parameter to `True` in order to be able to use the new behavior... which doesn't make sense at all. And changing the default value for `update` is disruptive as well.

The option that brings the least issues is to leave the `update` parameter but warn that it is not used: the VM will only be updated if there is a difference in the configuration. This fits more the Salt states spirit and in all cases the update doesn't stop the VM to update it: it tries as much as possible to apply live changes.

Reply:
> users have to set the deprecated parameter to True in order to be able to use the new behavior... which doesn't make sense at all. 

Why does this not make sense? This would allow this to be an opt in feature while not changing behavior for users for another couple releases so they are prepared.